.github/workflows/packages_publishing.yml

```diff
@@ -50,17 +50,17 @@ jobs:
       - name: Install dependencies
         run: npm install --no-audit --no-fund
 
-      - name: Set timestamp version 
+      - name: Set timestamp version
         if: ${{ env.SET_TIMESTAMP_VERSION == 'true' }}
-        run: npm run ts-script build/set-timestamp-version
+        run: npm run ts-node build/set-timestamp-version
 
       - name: Build npm packages
         env:
           BUILD_INTERNAL_PACKAGE: true
         run: npm run all:build
 
       - name: Build artifacts package
-        run: npm run ts-script build/make-artifacts-package
+        run: npm run ts-node build/make-artifacts-package
```

Comment:
Why not just `npx ts-node ...`? The behavior should be the same, npx calls bin packages from local node_modules if they exist

Reply:
You're right, no more need in script (I used it in local experiments)

Fixed.

Reply:
Remove it from package.json `scripts` field too please

Reply:
done